features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/Landing/Menus/RestAPIMenu.jsx

```diff
@@ -8,12 +8,30 @@ import SampleAPI from 'AppComponents/Apis/Listing/SampleAPI/SampleAPI';
 import Divider from '@material-ui/core/Divider';
 import Box from '@material-ui/core/Box';
 import Configurations from 'Config';
+import { useApiListContext } from 'AppComponents/Shared/ApiListContext';
+
+/**
+ * Verify whether the sample API has already been deployed.
+ * @param {(Array | null)} apisAndApiProducts list of APIs
+ * @returns {boolean} true if API already deployed/exists
+ */
+const verifyIfSampleDeployed = (apisAndApiProducts) => {
+    if (apisAndApiProducts && Array.isArray(apisAndApiProducts)) {
```

Comment:
Ahh I think we should have used APIs search here, otherwise, this could break if the sample API is on the next page with the pagination, isn't it?

Similar logic has been used when creating an API in default API form

Reply:
fixed with 1bc0c49